x/staking/client/cli/tx.go

```diff
@@ -172,11 +175,11 @@ func ValidatorJoinTxCmd() *cobra.Command {
 	cmd.Flags().String(FlagAmount, "0", "--amount=<amount>")
 	cmd.Flags().Uint64(FlagActivationEpoch, 0, "--activation-epoch=<activation-epoch>")
 
-	cmd.MarkFlagRequired(FlagBlockNumber)
-	cmd.MarkFlagRequired(FlagActivationEpoch)
-	cmd.MarkFlagRequired(FlagAmount)
-	cmd.MarkFlagRequired(FlagSignerPubkey)
-	cmd.MarkFlagRequired(FlagTxHash)
+	cmd.MarkFlagRequired(FlagBlockNumber)     //nolint
+	cmd.MarkFlagRequired(FlagActivationEpoch) //nolint
+	cmd.MarkFlagRequired(FlagAmount)          //nolint
+	cmd.MarkFlagRequired(FlagSignerPubkey)    //nolint
+	cmd.MarkFlagRequired(FlagTxHash)          //nolint
```

Comment:
I am ok with this but how about the following? It looks a bit better for me
```suggestion
	_ = cmd.MarkFlagRequired(FlagBlockNumber)
	_ = cmd.MarkFlagRequired(FlagActivationEpoch)
	_ = cmd.MarkFlagRequired(FlagAmount)
	_ = cmd.MarkFlagRequired(FlagSignerPubkey)
	_ = cmd.MarkFlagRequired(FlagTxHash)
```